tests/microos/one_line_checks.pm

```diff
@@ -14,8 +14,8 @@ use utils;
 
 sub run_rcshell_checks {
     # Check that system is using UTC timezone
-    my $timezone = get_var('FIRST_BOOT_CONFIG', '') =~ /cloud-init/ ? 'CEST' : 'UTC';
-    assert_script_run "date +\"%Z\" | grep -x $timezone";
+    my $timezone = get_var('FIRST_BOOT_CONFIG', '') =~ /cloud-init/ ? 'CE\(S?\)T' : 'UTC';
```

Comment:
I agree in removing brackets.
Not sure if single-quotes works, better double.
Moreover as proposal, to help parser's work and readability, you could enclose the condition in brackets:
```suggestion
    my $timezone = get_var('FIRST_BOOT_CONFIG', '') =~ (/cloud-init/) ? "CE\(S?\)T" : "UTC";
```

Reply:
Readability is very subjective, in my opinion this makes it worse. But that is only my opinion over here.